assets/figures/cash_payment.tex

```diff
@@ -0,0 +1,18 @@
+    
+\begin{tikzpicture}[scale = 0.67]
```

Comment:
General problem: At some point, we should remove the scale parameters from the figure source and specify the scale In the slide tex. No need to fix here. Let's do that at a later stage.

Reply:
I have added this to an overarching to do list on the slide style guidelines

Reply:
Here we go: Issue https://github.com/cifunibas/Bitcoin-Blockchain-Cryptoassets/issues/13